src/main/java/www/purple/mixxy/models/Comic.java

```diff
@@ -0,0 +1,53 @@
+package www.purple.mixxy.models;
+
+/**
+ * Created by Brian_Sabz on 4/5/16.
+ * 
+ * @author Brian_Sabz
+ */
+import com.googlecode.objectify.Ref;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Load;
+
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Index
+public class Comic {
+
+	@Id
+	public Long id;
+
+	// can refer to these Entities using the already fetched Comic, this solve
+	// performance problems
+	// See: https://github.com/objectify/objectify/wiki/Entities#load
+	@Load
+	public Ref<User> author;
+	@Load
+	public Ref<Comic> parent;
+
+	@Index
+	public String title;
+	public String description;
+	public List<String> tags;
+	public int version;
+
+	public Date createdAt;
+	public Date updatedAt;
```

Comment:
I know we'll need to query by dates (e.g. for latest comics), but I'm not sure if this should be an index or a filter.  I say leave it as is for now and index it if we need to.